### PR TITLE
Three new ways to link to external sites from authority pages and minor changes to help text

### PR DESCRIPTION
**Pull request by confirmordeny**
## Relevant issue(s)

(1) Adds links from public authority pages to:
(a) the Office for Students register of education providers; 
(b) OpenLEI; and
(c) WikiData.

(2) Minor changes to help text.

## What does this do?
See above.

## Why was this needed?
See above.

## Implementation notes
N/A

## Screenshots
N/A

## Notes to reviewer
N/A

### CODE REVIEW

**Review by gbp — CHANGES_REQUESTED** [review]
Nice. Good additions here.

Only thing is the comments will be outputted to the webpage too. So they will need to either removed or put between `<% ... %>` ERB template tags.

Personally I would remove the comments, rebase the branch on the current WDTK-theme master and [squash](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits) the commits so there is a single commit for each external site and use the comment as the commit message.

Happy to do this for you but maybe you would like to have a go at this first?

File: lib/views/help/glossary.html.erb [hunk 1]
```diff
@@ -138,6 +138,13 @@ Environmental Information Regulations 2004 (
 <dl>
 <dt id="gdpr">GDPR</dt>
 <dd>General Data Protection Regulation - a law about data protection and privacy. Following Brexit, the UK has its own version of GDPR. See also <a href="https://www.legislation.gov.uk/eur/2016/679/contents">the text of the General Data Protection Regulation</a>.</dd>
+</dl>
+ <dl>
+<dt id="github">GitHub</dt>
+<dd>GitHub is an online tool for software development and version control. 
+ Issues can be raised in GitHub where there are bugs in the site software or to suggest improvements or new features. 
+  The code behind the WhatDoTheyKnow website is worked on using GitHub. See also <a fre="#alaveteli>Alaveteli</a>.
```
Comment (garethrees) [inline lib/views/help/glossary.html.erb:146]: typo: `<a fre=`

File: lib/views/public_body/_more_info.html.erb [hunk 2]
```diff
@@ -41,6 +41,18 @@
   <% end %>
 <% end %>
 
+# A UKPRN is a UK Provider Reference Number.
```
Comment (garethrees) [inline lib/views/public_body/_more_info.html.erb:44]: These will be rendered to the HTML.

Comment in erb is:

```erb
<%
# foo bar baz…
%>
```

Comment (garethrees) [reply lib/views/public_body/_more_info.html.erb:44]: oops missed graemes note

File: lib/views/help/glossary.html.erb (hunk 1 above)
Comment (gbp) [reply lib/views/help/glossary.html.erb:146]: This is from a previous commit in a different PR which has already been merged so I have fixed in faf1be82990d86f176d20e2488507f75aebf2f08

Nice spot

**Comment by confirmordeny** [comment]
I am going to try to squash some commits.

**Pull request closed without merging**